### PR TITLE
chore: add "add image" button to empty state [f-5]

### PR DESCRIPTION
**Pull request by frederickfogerty**
ssia

Normal state:
image
Hover state:
image

### Stacked PR Chain: f-5
| PR | Title | Status |  Merges Into  |
|:--:|:------|:-------|:-------------:|
|#106|chore: refactor asset click handling to AssetBrowser ||-|
|#104|chore: reduce number of files in sidebar app story ||#106|
|#105|fix: render image in sidebar app at right size ||#104|
|#107|👉 chore: add "add image" button to empty state ||#105|
|#108|feat: add replace image button overlay ||#107|
|#109|chore: change image preview aspect raio in sidebar to be golden ratio ||#108|
|#110|chore: add user-select: none to button text to stop highlighting ||#109|
|#111|chore: add sidebar caption and ability to remove image ||#110|

### CODE REVIEW

File: frontend/src/components/SidebarApp.tsx [hunk 1]
```diff
@@ -2,7 +2,7 @@ import React from "react";
 import Imgix from "react-imgix";
 import "../styles/App.css";
 import { IImgixCustomAttributeValue } from "../types/imgixSF";
-import { AddImageIcon } from "./buttons/AddImageIcon";
+import { AddSvg } from "./icons/AddSvg";
```
Comment (luqven) [inline frontend/src/components/SidebarApp.tsx:5]: Just double checking, should we remove`AddImageIcon` button now from the repo?

File: frontend/src/components/SidebarApp.tsx [hunk 2]
```diff
@@ -16,7 +16,7 @@ export function App({ handleBreakoutOpen, value }: ISidebarAppProps) {
       <header className="App-header">
         <div>
           <div className={styles.imageWrapper}>
-            {value && (
+            {value ? (
```
Comment (luqven) [inline frontend/src/components/SidebarApp.tsx:19]: Nice change ➕

**Review by luqven — APPROVED** [review]
LGTM 👍🏼

File: frontend/src/components/SidebarApp.tsx (hunk 1 above)
Comment (frederickfogerty) [reply frontend/src/components/SidebarApp.tsx:5]: Yeah sure thing

**Pull request merged**